tools/ci_build/github/azure-pipelines/qnn-ep-nuget-packaging-pipeline.yml

```diff
@@ -224,15 +237,18 @@ jobs:
             $arm64_nupkg_unzipped_directory = [System.IO.Path]::Combine($arm64_nupkg_unzipped_directory_root, 'binaries', [System.IO.Path]::GetFileNameWithoutExtension($arm64_nuget_package))
             [System.IO.Compression.ZipFile]::ExtractToDirectory($arm64_nuget_package, $arm64_nupkg_unzipped_directory)
 
-            $arm64_runtime_path_old = [System.IO.Path]::Combine($arm64_nupkg_unzipped_directory, 'runtimes', 'win-arm64', 'native')
-            $arm64_runtime_path_new = [System.IO.Path]::Combine($x64_nupkg_unzipped_directory, 'runtimes', 'win-arm64', 'native')
+            $x64_runtime_path_old = [System.IO.Path]::Combine($x64_nupkg_unzipped_directory, 'runtimes', 'win-x64', 'native')
+            $x64_runtime_path_new = [System.IO.Path]::Combine($arm64_nupkg_unzipped_directory, 'runtimes', 'win-x64', 'native')
 
-            New-Item -Path $arm64_runtime_path_new -ItemType Directory
+            New-Item -Path $x64_runtime_path_new -ItemType Directory
 
-            Copy-Item ([System.IO.Path]::Combine($arm64_runtime_path_old, 'onnxruntime.dll'))                  $arm64_runtime_path_new
-            Copy-Item ([System.IO.Path]::Combine($arm64_runtime_path_old, 'onnxruntime.lib'))                  $arm64_runtime_path_new
-            Copy-Item ([System.IO.Path]::Combine($arm64_runtime_path_old, 'onnx_test_runner.exe'))             $arm64_runtime_path_new
-            Copy-Item ([System.IO.Path]::Combine($arm64_runtime_path_old, 'onnxruntime_perf_test.exe'))        $arm64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'onnxruntime.dll'))                  $x64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'onnxruntime.lib'))                  $x64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'onnx_test_runner.exe'))             $x64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'onnxruntime_perf_test.exe'))        $x64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'QnnCpu.dll'))                       $x64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'QnnHtp.dll'))                       $x64_runtime_path_new
+            Copy-Item ([System.IO.Path]::Combine($x64_runtime_path_old, 'QnnSaver.dll'))                     $x64_runtime_path_new
```

Comment:
are we missing some dll's to copy?
is the list supposed to match the list below in generate_nuspec_for_native_nuget.py ?